x-pack/plugins/actions/server/lib/mustache_renderer.ts

```diff
@@ -144,13 +144,6 @@ function escapeSlack(value: unknown): string {
   if (value == null) return '';
 
   const valueString = `${value}`;
-  // if the value contains * or _, escape the whole thing with back tics
```

Comment:
This feels like it's too broad of a change.  The reason we escape variable content containing `*` or `_` is to prevent Slack from treating these as **bold** or _italic_.  By removing this escaping completely, variables with those characters will end up causing probably unwanted style changes in the message.

I wonder if we can be a little more specific.  Could we check if the value is a URL (see if URL.parse() or whatever doesn't throw an error)?  If it **IS** a URL, do something different, otherwise do the existing thing (wrap the whole thing in back-tics).  

As an additional thought, I wonder if we should uri encode the URL while we're at it?  This would fix some [additional issues](https://github.com/elastic/kibana/issues/168728), but is somewhat _magical_, so my brain says no ...

Reply:
Resolved in this commit 1c76b5fe035e7c1795cf8f211dde05ae391ca0f5